### ISSUE TITLE
missing glyphs in adobe muse...

### ISSUE THREAD

**Issue by dayzed007** [FortAwesome/Font-Awesome#10433]
Unsure if this is a fontawesome issue or an adobe muse problem.

I had FA 4.3 installed and working fine.  Today at the start of a new project i discovered the availability of FA 4.7 and choose to upgrade;

1. Uninstalled 4.3 from mac system (font book)
2. Uninstalled 4.3 from muse webfonts manager
3. Installed 4.7 into mac system
4. Installed 4.7 into muse webfonts manager
5. Rebooted both Muse and Mac

Only now when I use FA in muse via the glyph panel, I see no previews, nor can I use it on screen.  Has anyone else experienced this problem?

Font-Awesome 4.7.0
Adobe Muse CC 2017
macOS Sierra 10.12.2

**Comment by tagliala** [comment]
Sorry, I can't help here

> muse webfonts manager

You tried both `.ttf` and `.otf`?

**Comment by dayzed007** [comment]
I tried primarily the .ttf since that's what worked for me with 4.3.
Since I posted my previous I ran the process again. Removed all instances of 4.7, went back to 4.3 and verified it all worked as before, then forward again to 4.7, but alas no improvement :-(

This seems to suggest that the problem falls with FA 4.7 itself and not Muse, since 4.3 works fine, as do all my other webfonts.

**Comment by tagliala** [comment]
The .ttf font is generated via [Font Squirrel](https://www.fontsquirrel.com/tools/webfont-generator). Or maybe Dave introduced some optimization specific for the .ttf format. Please give a try to the .otf format

**Comment by dayzed007** [comment]
Thanks, but still no luck :-(  There seems to be a disconnect of sorts between 4.7 and muse, I just can't imagine what.

I removed the TTF, and installed the OTF but still got the same results.  Just to cover my bases I tried fresh as before but with the same results.  For now I'm leaving font awesome off my system since it's of no use to me (or maybe I'll fall back to 4.3)

**Comment by tagliala** [comment]
@dayzed007 sorry that this doesn't work.

I'm leaving this open but I cannot help since I don't use adobe muse, maybe someone else could help

**Comment by dayzed007** [comment]
Just adding new information to this thread.  Someone shared the following instructions with me;

http://pixelosaur.com/blog/how-to-use-font-awesome-in-muse/

What has me confused about this is the fact that previously I didn't know about this complex way of using FA, and simply installed and used it as any other webfont in Muse.  But now it seems the recommendation is to use this complex system.

Did something drastic change between 4.3 and 4.7 which would cause Muse to have new requirements for the same product?

**Comment by joereplicator** [comment]
Yes: Font Awesome does create a bunh of errors in Adobe Muse. 

It is installed correctly:  http://prntscr.com/h0074l)
It displays correctly on the canvas: http://prntscr.com/h007ha
It publishes correctly: http://prntscr.com/h008jw
It also produces a TON of errors:  http://prntscr.com/h00884

Note there is no instance of a "." being used with the icons and there are no "bolded" icons.
After I removed the 6 instances of use of this icon the errors stopped. 
I checkd at all breakpoints and there are no instances of the font being used in bold. 
I spent an hour today with Adobe support and they refered this issue back to the makers of FA.

I am willing to share my Adobe Muse file should you wish to try it.
